tools/rflx_pylint_checker.py

```diff
@@ -0,0 +1,30 @@
+import astroid
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+
+
+class NamedBooleanArgumentsChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = "named-boolean-arguments"
+    priority = -1
+    msgs = {
+        "C0001": {
+            "Anonymous boolean argument",
+            "anon-bool-arg",
+            "Boolean arguments must be passed as keywords",
+        }
+    }
+
+    def visit_call(self, node: astroid.Call) -> None:
+        num_args = (len(node.args) if node.args else 0) + (
+            len(node.keywords) if node.keywords else 0
+        )
+        if num_args > 1:
+            for arg in node.args:
+                if isinstance(arg, astroid.Const) and arg.pytype() == "builtins.bool":
+                    self.add_message("C0001", node=arg)
```

Comment:
For which cases is `if num_args > 1:` needed here?

Reply:
I don't flag calls with only one parameter if they use a boolean literal as a positional argument. Hence `Set_Foo(True)` is OK. I found that these cases are often descriptive enough and `Set_Foo(Value=True)` would not be very readable.

Reply:
I see.